GSA/digitalgov.gov
# Update 2024-04-18-uswds-monthly-call-april-2024.md

Updating April 2024 monthly call page event to include script.

### Preview

[Link to Preview (After)](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/ss-april-uswds-mc-script/event/2024/04/18/uswds-monthly-call-april-2024/)

[Link to current state (Before)](https://digital.gov/event/2024/04/18/uswds-monthly-call-april-2024/)

### Solution

### How To Test

1. Compare the preview to the April 2024 monthly call script to make sure all content is accounted for, including formatting
2. Ensure that accordion component works.
---

### Dev Checklist
